NuRadioMC/utilities/Veff.py

```diff
@@ -557,7 +321,13 @@ def get_Veff_single(filename, trigger_names, trigger_names_dict, trigger_combina
                     triggered = triggered & (np.array(fin[f'station_{station:d}/ray_tracing_reflection'])[..., max_amps, 0][:, 0] == values['n_reflections'])
 
             triggered = remove_duplicate_triggers(triggered, fin['event_group_ids'])
-            Veff = V * np.sum(weights[triggered]) / n_events
+            Veff = volume_proj_area * np.sum(weights[triggered]) / n_events
```

Comment:
I guess this depends on how exactly effective volume is defined, but do we have to divide by number of events or by the sum of all weights? I.e. is absorption in the earth considered part of the effective volume? Just asking to be sure

Reply:
you always divide by the number of events. If Earth attenuation is considered the nominator is the sum of the triggered events times their weight.